### ISSUE TITLE
Doesn't handle [white]spaces correctly in paths

### ISSUE THREAD

**Issue by Tarrasch** [kennethreitz/autoenv#40]
While I know it's a terrible practice to have spaces in the name of your folders, it should still work. Here is a "test case" demonstrating that it doesn't work for me.

```
~ % mkdir 'hello world'
~ % cd hello\ world 
hello world % echo 'echo You should print this' > .env
```
And now I get some weird behaviour:
```
hello world % cd .
shasum: world/.env: 
autoenv:
autoenv: WARNING:
autoenv: This is the first time you are about to source world/.env:
autoenv:
autoenv:     --- (begin contents) ---------------------------------------
sed: can't read world/.env: No such file or directory
autoenv:
autoenv:     --- (end contents) -----------------------------------------
autoenv:
autoenv: Are you sure you want to allow this? (y/N) y
shasum: world/.env: 
autoenv_check_authz_and_run:source:22: no such file or directory: world/.env
shasum: /home/arash/hello: 
autoenv:
autoenv: WARNING:
autoenv: This is the first time you are about to source /home/arash/hello:
autoenv:
autoenv:     --- (begin contents) ---------------------------------------
sed: can't read /home/arash/hello: No such file or directory
autoenv:
autoenv:     --- (end contents) -----------------------------------------
autoenv:
autoenv: Are you sure you want to allow this? (y/N) y
shasum: /home/arash/hello: 
autoenv_check_authz_and_run:source:22: no such file or directory: /home/arash/hello
```

Also. Thank you so much for this awesome tool!!

**Comment by harrislapiroff** [comment]
I am also having this issue! +1 to the PR above!

**Comment by Tarrasch** [comment]
@harrislapiroff, I've abandaoned this plugin and went ahead and made my own much better version of it. https://github.com/Tarrasch/zsh-autoenv 

Closing since it was fixed apparently in #66
